services/movie_session.py

```diff
@@ -0,0 +1,52 @@
+from django.db.models import QuerySet
+
+from db.models import MovieSession
+
+
+def create_movie_session(
+        movie_show_time: int,
+        movie_id: int,
+        cinema_hall_id: int
+) -> QuerySet:
+    return MovieSession.objects.create(
+        show_time=movie_show_time,
+        movie_id=movie_id,
+        cinema_hall_id=cinema_hall_id,
+    )
+
+
+def get_movies_sessions(session_date: list[str] = None) -> QuerySet:
+    if session_date:
+        return MovieSession.objects.filter(show_time__date=session_date)
+    return MovieSession.objects.all()
+
+
+def get_movie_session_by_id(movie_session_id: int) -> QuerySet:
+    return MovieSession.objects.get(id=movie_session_id)
+
+
+def update_movie_session(
+        session_id: int,
+        show_time: list[str] = None,
+        movie_id: list[str] = None,
+        cinema_hall_id: list[str] = None
+) -> QuerySet:
+    try:
+        movie_session = MovieSession.objects.get(id=session_id)
+    except MovieSession.DoesNotExist:
+        raise ValueError(f"Movie session with id {session_id} does not exist")
+
+    if show_time is not None:
+        movie_session.show_time = show_time
+    if movie_id is not None:
+        movie_session.movie_id = movie_id
+    if cinema_hall_id is not None:
+        movie_session.cinema_hall_id = cinema_hall_id
+
+    movie_session.save()
+
+    return movie_session
+
+
+def delete_movie_session_by_id(session_id: int) -> QuerySet:
+    return MovieSession.objects.filter(id=session_id).delete()
```

Comment:
.delete()
This method returns the number of objects deleted and a dictionary with the number of deletions per object type.
https://docs.djangoproject.com/en/4.2/ref/models/instances/#django.db.models.Model.delete:~:text=This%20method%20returns%20the%20number%20of%20objects%20deleted%20and%20a%20dictionary%20with%20the%20number%20of%20deletions%20per%20object%20type.